packages/ckeditor5-mention/src/mention.js

```diff
@@ -131,6 +131,45 @@ export default class Mention extends Plugin {
  * @default [ 13, 9 ] // [ Enter, Tab ]
  */
 
+/**
+ * The configuration of the custom number of visible mentions.
+ *
+ *		ClassicEditor
+ *			.create( editorElement, {
+ *				plugins: [ Mention, ... ],
+ *				mention: {
+ *	 				dropdownLimit: 20,
+ *					feeds: [
+ *						{ ... }
+ *						...
+ * 					]
+ *				}
+ *			} )
+ *			.then( ... )
+ *			.catch( ... );
+ *
+ *		ClassicEditor
+ *			.create( editorElement, {
+ *				plugins: [ Mention, ... ],
+ *				mention: {
+ *	 				dropdownLimit: Infinity,
+ *					feeds: [
+ *						{ ... }
+ *						...
+ * 					]
+ *				}
+ *			} )
+ *			.then( ... )
+ *			.catch( ... );
+ *
+ * Customizing the number of visible mentions allows you to specify how many available elements will users be able to see
+ * in the dropdown list. You can specify any number you'll see fit. For example, in the snippets above you will find the
+ * dropdownLimit set to `20` and `Infinity` (this will result in showing all available mentions).
```

Comment:
Please move this description before the code samples - without context it isn't clear what to look at in them.